client/src/screens/Product.js

```diff
@@ -208,13 +209,23 @@ export default () => {
     setQuestion(questionDescription)
   }
 
+  const setQuestionForm = () => {
+    setIsModalOpen(true)
+    setModalContentType('question')
+  }
+
   const setReportForm = (item, fixedquestionIndex, answerIndex) => {
     setIsModalOpen(true)
     setModalContentType('report')
 
     setReportItem({ fixedQandAsId: item._id, fixedquestionIndex, answerIndex })
   }
 
+  const closeModal = () => {
+    setIsModalOpen(false)
+    setQuestion('')
```

Comment:
question is not string type

Reply:
When we setQuestion({}) here, then error occurs!

Reply:
how about null?

Reply:
let me

Reply:
umm buggy
only "" works..